src/commands/make_proposal.rs

```diff
@@ -37,7 +37,11 @@ pub struct MakeProposalOpts {
     ///     }
     /// )'
     #[clap(long)]
-    proposal: String,
+    proposal: Option<String>,
+
+    /// Path to the WASM file to be installed onto the target canister.
```

Comment:
The doc here is wrong.

And only one of `proposal` and `proposal_path` can be set here.
I would suggest to add a `conflicts_with` attribute.
An example can be found [here](https://github.com/dfinity/sdk/blob/298bc23449f172ae51f44f19cb48df0b64d31616/src/dfx/src/commands/deploy.rs#L72).